src/azul/plugins/repository/canned/__init__.py

```diff
@@ -2,13 +2,10 @@
 This repository plugin allows reading from a canned staging area like the one in
 the GitHub repo https://github.com/HumanCellAtlas/schema-test-data .
 
-NOTE: Use of this plugin requires a GitHub personal access token specified in an
-environment variable `GITHUB_TOKEN`. See:
-https://github.com/DataBiosphere/hca-metadata-api#github-credentials
```

Comment:
Does this mean we no longer need to configure GitHub tokens for our deployments? Or are they still needed for something else?